### ISSUE TITLE
Store Q&As in Hugging Face

### ISSUE THREAD

**Issue by dominic0df** [amosproj/amos2024ss08-cloud-native-llm#33]
## User story
1. As data engineer
2. I want / need to store my generated Q&A pairs in my Q&A dataset
3. So that I will be able to perform machine learning on the generated Q&A set afterwards

## Acceptance criteria
* The Q&A pairs generated by the Q&A Generator are stored together in our Q&A dataset in Hugging Face

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
